db/migrate/20190214125845_add_comments_to_miq_tasks_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddCommentsToMiqTasksTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :miq_tasks, "ManageIQ background tasks"
+
+    change_column_comment :miq_tasks, :context_data, "Serialized metadata that may contain more detailed information about the task."
+    change_column_comment :miq_tasks, :created_on, "The timestamp the task was added to the app inventory."
+    change_column_comment :miq_tasks, :id, "The internal database ID."
+    change_column_comment :miq_tasks, :identifier, "The resource this task is related to in '#{klass}:#{id}' format."
+    change_column_comment :miq_tasks, :message, "Message that describes the reason for the current status."
+    change_column_comment :miq_tasks, :miq_server_id, "ID of the ManageIQ server that the task belongs to."
+    change_column_comment :miq_tasks, :name, "The name of the task. Typically doubles as a description."
```

Comment:
there is no `description` column in `miq_task` table, I am not sure where name of task doubles as a description

Reply:
In practice what I'm seeing is stuff like "Generate Widget: 'EVM: Recently Discovered Hosts'". That's really more of a description than a name, at least as I typically think of it.

Reply:
Ohh, you meant it usually used to put descriptive name

Reply:
At least some of our tables have separate `name` and `description` fields, but here it seems to be used interchangeably.